patches/openwrt/0032-ar71xx-support-for-Meraki-MR12-and-MR62.patch

```diff
@@ -0,0 +1,306 @@
+From: Nils Schneider <nils@nilsschneider.net>
+Date: Sat, 4 Apr 2015 17:23:39 +0200
+Subject: ar71xx: support for Meraki MR12 and MR62
+
+diff --git a/target/linux/ar71xx/base-files/etc/diag.sh b/target/linux/ar71xx/base-files/etc/diag.sh
+index 6aac86a..cd9b53c 100755
+--- a/target/linux/ar71xx/base-files/etc/diag.sh
++++ b/target/linux/ar71xx/base-files/etc/diag.sh
+@@ -100,6 +100,9 @@ get_status_led() {
+ 	ls-sr71)
+ 		status_led="ubnt:green:d22"
+ 		;;
++	mr12)
++		status_led="mr12:green:power"
++		;;
+ 	mr600)
+ 		status_led="mr600:orange:power"
+ 		;;
+diff --git a/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds b/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds
+index 315a9fd..8d335ee 100755
+--- a/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds
++++ b/target/linux/ar71xx/base-files/etc/uci-defaults/01_leds
+@@ -158,6 +158,14 @@ hornet-ub)
+ 	ucidef_set_led_usbdev "usb" "USB" "alfa:blue:usb" "1-1"
+ 	;;
+ 
++mr12)
++	ucidef_set_led_netdev "wan" "WAN" "mr12:green:wan" "eth0"
++	ucidef_set_led_wlan "wlan1" "WLAN1" "mr12:green:wifi1" "phy0assoc"
++	ucidef_set_led_wlan "wlan2" "WLAN2" "mr12:green:wifi2" "phy0assoc"
++	ucidef_set_led_wlan "wlan3" "WLAN3" "mr12:green:wifi3" "phy0assoc"
++	ucidef_set_led_wlan "wlan4" "WLAN4" "mr12:green:wifi4" "phy0tpt"
```

Comment:
I think this should be changed to use rssileds like some other models with multiple WLAN LEDs do.